codespell_lib/data/dictionary.txt

```diff
@@ -7889,10 +7890,22 @@ datsets->datasets
 daty->data, date,
 daugher->daughter
 DCHP->DHCP
+dcok->dock
+dcoked->docked
 dcoker->docker
+dcokerd->dockerd
```

Comment:
Is dockerd a reference to the Docker software package? I'm not sure it belongs in the "regular" dictionary? ping @larsoner

Reply:
Yes @lurch it is a reference to Docker.

Reply:
```suggestion
dcokerd->dockerd, docker,
```

Would probably the easy solution.

Reply:
> Is `dockerd` a reference to the Docker software package? I'm not sure it belongs in the "regular" dictionary? ping @larsoner

It only matters if the typo is in the main dictionary. What we don't want to do is have:
`dockerd->docker`

In the main dictionary, everyone goes and runs that on their codebases as it's enabled by default and suddenly everything breaks.

Having `dokerd->dockerd` and `doker->docker` are both fine though as they won't break anything.

Reply:
> In the main dictionary, everyone goes and runs that on their codebases as it's enabled by default and suddenly everything breaks.

Very good point.